ipatests/test_integration/test_installation.py

```diff
@@ -567,3 +567,55 @@ def test_KRA_install_after_cert_renew(self):
         self.master.run_command(['kinit', 'admin'], stdin_text=passwd)
         cmd = self.master.run_command(['ipa-kra-install', '-p', dm_pass, '-U'])
         self.master.run_command(['systemctl', 'start', 'chronyd'])
+
+
+class TestInstallReplicaAgainstSpecificServer(IntegrationTest):
+    """Installation of replica against a specific server
+
+    Test to check replica install against specific server. It uses master and
+    replica1 without CA and having custodia service stopped. Then try to
+    install replica2 from replica1 so that replica2 will fetch secrets from
+    master as custodia service is not running on replica1.
```

Comment:
It's not clear to me what should be the expected behavior in this case. Replica2 / master communications are prevented by the firewall, so should we expect the test to fail?

Reply:
Please see https://github.com/freeipa/freeipa/pull/2331#issuecomment-419455394

Reply:
Sorry but I still don't get your point:
- replica2 is installed with setup_ca = True (default value in tasks.install_replica) but there is no CA on replica1 => the installer should use master because it needs a CA.
- replica2 / master communications are blocked
So the test should be expecting a failure.

Reply:
Please update the test description. The installation is expected to fail as replica1 does not provide a CA and was forced with --server param.